code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -182,12 +191,12 @@
 			if(!vampire_can_affect_target(L, 0, affect_ipc = TRUE))
 				continue
 
-			L.Weaken(8)
-			L.stuttering = 20
-			L.confused = 10
 			to_chat(L, SPAN_DANGER("You are blinded by [src]'s glare!"))
```

Comment:
this should be below the if, and there should be a message that you resisted the glare in an else